serving/docs/lmi/tutorials/trtllm_aot_tutorial.md

```diff
@@ -110,8 +110,8 @@ aws s3 cp $MODEL_REPO_DIR s3://YOUR_S3_FOLDER_NAME/ --recursive
 **Note:**  After uploading model artifacts to s3, you can just update the model_id(env var or in `serving.properties`) to the newly created s3 url with compiled model artifacts and use the same rest of the environment variables or `serving.properties`  when deploying on SageMaker. Here, you can check the [tutorial](https://github.com/deepjavalibrary/djl-demo/blob/master/aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb) on how to run inference using TensorRT-LLM DLC.  Below snippet shows example updated model_id.
 
 ```
-OPTION_MODEL_ID=s3://YOUR_S3_FOLDER_NAME
-OPTION_TENSOR_PARALLEL_DEGREE=8
+HF_MODEL_ID=s3://YOUR_S3_FOLDER_NAME
```

Comment:
While this works functionally,  what we are passing is not `HF_MODEL_ID`, we are passing TRT-LLM compiled artifacts. Not sure if this is very intuitive for customers, we should probably use a more generic name.